cinema_service/settings.py

```diff
@@ -52,6 +53,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
Consider making the inclusion of `debug_toolbar.middleware.DebugToolbarMiddleware` in `MIDDLEWARE` conditional based on the `DEBUG` setting to avoid unnecessary overhead in production.